src/utils/accountPaymentSubscribe.ts

```diff
@@ -0,0 +1,29 @@
+import StellarSDK, { ServerApi } from "stellar-sdk";
+
+const accountPaymentSubscribe = (
+  publicKey: string,
+  onUpdate: (payment: ServerApi.PaymentOperationRecord) => Promise<void>,
```

Comment:
Also, I think you can do this here
```suggestion
  onUpdate: (payment: PaymentOperationRecord) => Promise<void>,
```